driver/docker-container/driver.go

```diff
@@ -44,17 +44,18 @@ type Driver struct {
 
 	// if you add fields, remember to update docs:
 	// https://github.com/docker/docs/blob/main/content/build/drivers/docker-container.md
```

Comment:
Looks like this PR might've missed updating these docs :eyes:

Reply:
It's here :-) https://github.com/docker/docs/pull/19355